src/Vts/MonteCarlo/Extensions/StatisticsExtensions.cs

```diff
@@ -8,41 +8,46 @@ namespace Vts.MonteCarlo.Extensions
     public static class StatisticsExtensions
     {
         /// <summary>
-        /// Method to determine statistics about how the photon died.
+        /// Method to determine statistics about how the photon died.  These states
+        /// are typically mutually exclusive, however there are occurrences of photons
+        /// that are killed by RR at start of track that crosses boundary
+        /// and gets that state set as well but does not tally.
+        /// It is for this reason the check for RR is first.
         /// </summary>

```

Comment:
Do you think we should write out Russian roulette in this documentation?

Reply:
Sounds good.  Thanks for suggestion.